index.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 var clearDirectory = require('./clear').clearDirectory;
+var log = require('fancy-log');
+var colors = require('ansi-colors');
+var PluginError = require('plugin-error');
```

Comment:
Is this plugin error working with gulp 3?

Reply:
I don't know for sure, but I guess it is. The `plugin-error` package was extracted from gulp3 into a seperate project. The new package doesn't has any dependencies to gulp 3 so I guess it's just simply implemented twice if you use gulp 3, but this shoudn't be an issue.